src/maptemplate.c

```diff
@@ -4996,22 +5025,29 @@ int msReturnOpenLayersPage(mapservObj *mapserv) {
   if (tmpUrl)
     openlayersUrl = (char *)tmpUrl;
 
+  /* now do the same for the MS_OPENLAYERS_CSS_URL */
+  tmpUrl = msGetConfigOption(mapserv->map, "MS_OPENLAYERS_CSS_URL");
+  if (tmpUrl == NULL)
+    tmpUrl = CPLGetConfigOption("MS_OPENLAYERS_CSS_URL", NULL);
+
+  if (tmpUrl)
+    openlayersCssUrl = (char *)tmpUrl;
```

Comment:
```suggestion
    openlayersCssUrl = tmpUrl;
```

Reply:
This was copied from the JS setting, so I presume that should also be updated?

```
  if (tmpUrl)
    openlayersUrl = (char *)tmpUrl;
```

Reply:
> This was copied from the JS setting, so I presume that should also be updated?
> 
> ```
> 
> ```

yes, you can always (I shouldn't say "always" as I'm pretty sure someone could invent on purpose a corner use case in C++ where that wouldn't be true) safely remove such const related casts as soon as the compiler doesn't warn when they're removed